setup.py

```diff
@@ -32,11 +32,11 @@
     "Programming Language :: Python :: 3.7",
 ]
 
-ENTRY_POINTS = {
-    "console_scripts": [
-        "silicone-explore-quantiles-rolling-windows = silicone.cli:plot_emission_correlations_quantile_rolling_windows_cli"
-    ]
-}
+# ENTRY_POINTS = {
```

Comment:
Is there a particular reason you're commenting this out rather than deleting it? You'd have to re-import the thing you're referencing anyway if you want to re-insert it.

Reply:
setting up entry points isn't something I know how to do off the top of my head so if I just leave the code I can adapt it/when we bring a CLI back in (rather than having to look up how it works again). happy to delete if desired though (the 2 minute look up is a relatively small price)

Reply:
you can leave it if it saves time. I've OKed the changes, just generally better practice not to do too much commenting out.

Reply:
Yep I fully agree, let's just get rid of it then (can always bring back)